_admin/art.php

```diff
@@ -22,11 +22,11 @@
             </div>
         </div>
         <div class="row">
-            <a href="https://secure.burningflipside.com/register/api/v1/art?fmt=xml">Data Dump Of Doom (.xml)</a> |
-            <a href="https://secure.burningflipside.com/register/api/v1/art/*/artLead?fmt=csv">Art Leads (.csv)</a> |
-            <a href="https://secure.burningflipside.com/register/api/v1/art/*/soundLead?fmt=csv">Sound Leads (.csv)</a> |
-            <a href="https://secure.burningflipside.com/register/api/v1/art/*/safetyLead?fmt=csv">Safety Leads (.csv)</a> |
-            <a href="https://secure.burningflipside.com/register/api/v1/art/*/cleanupLead?fmt=csv">Cleanup Leads (.csv)</a>
+            <a href="'.$page->secureUrl.'/register/api/v1/art?fmt=xml">Data Dump Of Doom (.xml)</a> |
+            <a href="'.$page->secureUrl.'/register/api/v1/art/*/artLead?fmt=csv">Art Leads (.csv)</a> |
+            <a href="'.$page->secureUrl.'/register/api/v1/art/*/soundLead?fmt=csv">Sound Leads (.csv)</a> |
+            <a href="'.$page->secureUrl.'/register/api/v1/art/*/safetyLead?fmt=csv">Safety Leads (.csv)</a> |
+            <a href="'.$page->secureUrl.'/register/api/v1/art/*/cleanupLead?fmt=csv">Cleanup Leads (.csv)</a>
```

Comment:
I'd rather just make these relative links (i.e. ../api). That seem like a good idea to you?